### ISSUE TITLE
Wifi "Not available"

### ISSUE THREAD

**Issue by ZONEXIEO** [ev3dev/ev3dev#1501]
I recently bought a wireless adapter i plugged it into my ev3 robot and its still write there "Wifi not available", what should i do to make it works? i followed this video: https://www.youtube.com/watch?v=Op3lDRRN-Ck
and its seems like its works for him, please respond fast as possible.

**Comment by dlech** [comment]
Please see https://github.com/ev3dev/ev3dev/wiki/USB-Wi-Fi-Dongles for a list of supported and known working Wi-Fi dongles.

**Comment by ZONEXIEO** [comment]
can i add a adapter that isnt in the list? there is any instruction to do that?

**Comment by dlech** [comment]
The device in the video is already on the list. You probably have the v2 which is already described in the *Edimax Alert* section of that page.

**Comment by ZONEXIEO** [comment]
No, i dont have same adapter like its, i have d-link n300 Nano usb adapter

**Comment by ZONEXIEO** [comment]
I just followed about whats he doing

**Comment by dlech** [comment]
Then feel free to list it under the not working adapters on that page. Be sure to include the USB vendor and product ID from `lsusb`.

**Comment by ZONEXIEO** [comment]
Ok how long its can take to be added to the working list?

**Comment by ZONEXIEO** [comment]
D-Link Wireless N300 Nano USB Adapter

**Comment by dlech** [comment]
First, you need to know what the VID and PID is for the device (often the same name is used for multiple chipsets so just the name on the box doesn't help much), then you have to see if there are any existing Linux drivers for it, then you have to compile the driver against the ev3dev kernel. There are some other issues here where people have done that.

**Comment by ZONEXIEO** [comment]
USB\VID_0BDA&PID_818B&REV_0200
USB\VID_0BDA&PID_818B

this is what i found, https://www.realtek.com/en/component/zoo/category/rtl8192eu-software

**Comment by ZONEXIEO** [comment]
Its not for linux only windows so i cant continue

**Comment by ZONEXIEO** [comment]
how much time its will take to add it? to support ev3
